eo-maven-plugin/src/main/java/org/eolang/maven/latex/LatexTemplate.java

```diff
@@ -63,7 +61,16 @@ public String asString() {
                     new ResourceOf("org/eolang/maven/latex/latex-template.txt")
                 )
             ).asString(),
-            this.code
+            this.extractedContent()
         );
     }
+
+    private String extractedContent() {
```

Comment:
@Suban05 it's not clear for why exactly this regular expression is used here. Could you please add some documentation to the method here explaining why this method exists and what it does